src/plugins/initiatives/declaration.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import deepFreeze from "deep-freeze";
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType, EdgeType} from "../../analysis/types";
+import {NodeAddress, EdgeAddress} from "../../core/graph";
+
+export const nodePrefix = NodeAddress.fromParts(["sourcecred", "initiatives"]);
+export const edgePrefix = EdgeAddress.fromParts(["sourcecred", "initiatives"]);
+
+export const initiativeNodeType: NodeType = deepFreeze({
+  name: "Initiative",
+  pluralName: "Initiatives",
+  prefix: NodeAddress.append(nodePrefix, "initiative"),
+  defaultWeight: 1,
+  description:
+    "An initiative, formatted as a Discourse wiki topic using a fixed template.",
+});
+
+/*
+	Note on the forward and backward naming convention.
+	It follows the core/graph.js documentation to use
+	a <subject> <verb> <object> format to figure out
+	the directionality.
+*/
+
+/**
+ * A Discourse topic (src) TRACKS (verb) an initiative (dst).
```

Comment:
As discussed [here](https://discourse.sourcecred.io/t/write-the-initiatives-plugin/269/6?u=decentralion), I don't think we need a separate edge for this--we can consider the Discourse topic which defines an initiative to be an implicit contribution towards that initiative.

Reply:
I've removed the edge type for trackers. Instead the idea is trackers will be added as a contribution. Using the editor to lower the weights in cases where the tracker doesn't provide much value in the form of discussion for example.